assets/airlines/2.json

```diff
@@ -0,0 +1,54 @@
+{
+    "icao": "2",
+    "name": "General Aviation Guernsey",
+    "callsign": {
+        "name": "Two",
+        "callsignFormats": [
+            "####"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["BE36", 5],
+            ["C172", 5],
+            ["C182", 5],
+            ["C208", 1],
+            ["C208", 2],
```

Comment:
C208, C510, C550, C750 are all duplicated in the default fleets of all of these airlines?

Reply:
Yeah, they're duplicated but I thought due to the number of files in this PR it wouldn't be necessary to go through all of them again

Reply:
Oh yes, I messed that up, no problem I'll fix it